Mathlib/Algebra/Star/SelfAdjoint.lean

```diff
@@ -156,6 +156,10 @@ theorem conjugate {x : R} (hx : IsSelfAdjoint x) (z : R) : IsSelfAdjoint (z * x
 theorem conjugate' {x : R} (hx : IsSelfAdjoint x) (z : R) : IsSelfAdjoint (star z * x * z) := by
   simp only [isSelfAdjoint_iff, star_mul, star_star, mul_assoc, hx.star_eq]
 
+@[aesop 90% apply]
+theorem conjugate_of_isSelfAdjoint {x : R} (hx : IsSelfAdjoint x) {z : R} (hz : IsSelfAdjoint z) :
```

Comment:
This is why I didn't like using `conjugate` in the names (although everywhere else I think we managed to get it to something that is sufficiently clear): `conjugate` is about the (right) action `x <• z := star z * x * z`. Without the `star` it's not an action, and the names could get confusing. My solution to this to keep the `conjugate` naming is to use `self` where context doesn't dictate what the theorem statement should be.
```suggestion
theorem conjugate_self {x : R} (hx : IsSelfAdjoint x) {z : R} (hz : IsSelfAdjoint z) :
```